README.md

```diff
@@ -1,11 +1,164 @@
+1. [Usage](#Usage)
+2. [Installation](#Installation)
+3. [Deployment](#Deployment)
+4. [Development](#Development)
+5. [Challenges](#Challenges)
+6. [Improvements](#Improvements)
+7. [Next Steps](#Next-Steps)
+8. [Authors](#Authors)
+9. [Acknowledgements](#Acknowledgements)
+
+## Usage
+
+This project was deployed to Heroku in December of 2020.
+
+The link to the Social Connect app can be found here: http://social-connect-app.herokuapp.com/
+
+## Installation
+
+**Prerequisites**
+
+- Install Node.js
+- Version >= 10.18 (Includes npm and npx)
+- Install create-react-app (optional)
+
+**How to run**
+
+1. Clone and enter the github repo: https://github.com/CSC398/social-connect
+2. Switch to `master`branch
+3. Install node modules using ` npm install`
+4. Build a production version of the app with `npm run build`
+5. Start the node.js server with ` node server.js`
+6. Navigate to `http://localhost:PORT`on web browser
+
+```
+ $ npm run build
+
+> social-connect@1.0.0 build /social-connect
+> react-scripts build
+
+Creating an optimized production build...
+Compiled without warnings.
+
+File sizes after gzip:
+
+226.99 KB build/static/js/2.a5df9fc1.chunk.js
+3.98 KB build/static/js/main.5fcda821.chunk.js
+777 B build/static/js/runtime-main.d1e23670.js
+
+The project was built assuming it is hosted at /.
+You can control this with the homepage field in your package.json.
+
+The build folder is ready to be deployed.
+You may serve it with a static server:
+
+npm install -g serve
+serve -s build
+
+Find out more about deployment here:
+
+https://cra.link/deployment
+```
+```
+$ node server.js
+Server started on port 5000
+```
+## Deployment
+
+**Prerequisites**
+1. Clone directory
+`$ git clone https://github.com/CSC398/social-connect/`
+2. cd into directory
+`$ cd social-connect`
+3. Switch to master branch
+`$ git checkout master`
+
+**How to deploy**
+1. Create a Heroku account
+2. Install Heroku Command Line Interface (CLI)
+`$ brew install heroku/brew/heroku`
+3. Login to Heroku through your terminal 
+`$ heroku login`
+4. Create an app on Heroku with the create-react-app buildpack 
+`$ heroku create <app_name> --buildpack https://github.com/mars/create-react-app-buildpack`
+5. Deploy your app 
+`$ git push heroku master`
+6. Scale your app
+`$ heroku ps:scale web=1`
+7. Open your app
+`$ heroku open`
+
+## Development
+
+Social Connect is built using `React.js` and `CSS makestyles` styling to create a functional user interface for gameplay.
```

Comment:
not `CSS makestyles`. We used `Material UI` for styling

Reply:
Oops, typo. Fixed!